wildfly-jakartaee-webapp-archetype/src/main/resources/archetype-resources/README.txt

```diff
@@ -15,21 +15,57 @@ a persistence unit "${rootArtifactId}PersistenceUnit" which uses the JakartaEE d
 In production environment, you should define a database in WildFly config and point to this database
 in "persistence.xml".
 
+If you change the datasource name and use provisioning (profile "provision" or "arq-provisioned", see below),
+you have to update the provisioning config.
+
 If you don't use entity beans, you can delete "persistence.xml".
 ==========================
 
 Jakarta Faces:
 The web application is prepared for Jakarta Faces 4.0 by bundling an empty "faces-config.xml" in "src/main/webapp/WEB-INF".
 In case you don't want to use Jakarta Faces, simply delete this file and "src/main/webapp/beans.xml".
+
+==========================
+Provisioning:
+The project defines two profiles "provision" and "arq-provisioned" that create a provisioned WildFly server (in "target/server" or "target/server_arquillian")
+by auto discovering features necessary to run this application.
+
+The profile "provision" is meant to build to build a server that contains this web application as deployment.
+The profile "arq-provisioned" also builds a server, then runs t he arquillian tests. This profile does not add the deployment, so that the arquillian
+tests can deploy the app themselves enriched with additional test classes.
+
+As "persistence.xml" uses the datasource "java:comp/DefaultDataSource", Glow must be configured to create 
+the datasource in the WildFly server config. Currently, the add-on "create default H2 database" is configured:
```

Comment:
The sentence "Currently, the add-on "create default H2 database" is configured" seems strange. "the default H2 datasource" perhaps?

Reply:
I applied your suggestion.